File: PHP/Runway.php
<?php

require 'LatLon.php';

class Runway
{
        public $airportCode = "";
        public $runwayCode = "";
        public $alt = 0.0;
        public $magHeading = 0.0;
        public $trueHeading = 0.0;
        public $centerLatLon = NULL;

        function Runway($airportCode, $alt, $runwayCode, $magHeading, $trueHeading, $lat, $lon)
        {
            $this->airportCode = $airportCode;
            $this->alt = $alt;
            $this->runwayCode = $runwayCode;
            $this->magHeading = $magHeading;
            $this->trueHeading = $trueHeading;
            $this->centerLatLon = new LatLon($lat, $lon);
        }
}

?>

File: Python/Runway.py
from LatLon import LatLon


class Runway(object):
    ''' Instance Variables '''
    airportCode, runwayCode = '', ''
    alt, magHeading, trueHeading = 0.0, 0.0, 0.0
    centerLatLon = None


    def __init__(self, airportCode, alt, runwayCode, magHdg, trueHdg, lat, lon):
        self.airportCode = airportCode
        self.alt = alt
        self.runwayCode = runwayCode
        self.magHeading = magHdg
        self.trueHeading = trueHdg
        self.centerLatLon = LatLon(lat, lon)
# end class Runway